TombEngine/Game/Lara/lara_struct.h

```diff
@@ -613,7 +615,7 @@ enum LaraAnim
 	LA_LADDER_LEFT_CORNER_INNER_START = 367,				// Ladder around inner left corner
 	LA_LADDER_LEFT_CORNER_INNER_END = 368,
 	LA_LADDER_RIGHT_CORNER_INNER_START = 369,				// Ladder around inner right corner
-	LA_LADDER_RIGHT_CORNER_INNER_END = 370,					// TODO: Remove.
```

Comment:
Why `LA_LADDER_RIGHT_CORNER_INNER_END` was renamed, but `LA_LADDER_LEFT_CORNER_INNER_END` remained? Why it's even renamed, does it have any relation to this pull request?